sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -130,15 +130,15 @@ export class ApplicationControllerService {
    */
   async transformToApplicationForAESTDTO(
     application: Application,
-  ): Promise<GetApplicationBaseDTO> {
+  ): Promise<GetApplicationBaseAPIOutDTO> {
     return {
       data: application.data,
       id: application.id,
       applicationStatus: application.applicationStatus,
       applicationNumber: application.applicationNumber,
       applicationFormName: application.programYear.formName,
       applicationProgramYearID: application.programYear.id,
-    } as GetApplicationBaseDTO;
+    } as GetApplicationBaseAPIOutDTO;
```

Comment:
actually, we can remove this `as GetApplicationBaseAPIOutDTO;` unless we are forcing any object to be a type